### PR TITLE
Fix propogation of timing info to GIFVisualizer

### LINKED ISSUE #766: Fix GIF timestamps when full simulation is not run.
The GIF timestamps are not correct if you don't run for 10 ns, we should fix this by adding in the option to specify total time or parameters to reconstruct it to the class.

### PR DESCRIPTION
**Pull request by hmacdope**
## Description
Provide a brief description of the PR's purpose here.
Fixes #766 

## Developers certificate of origin
- [X] I certify that this contribution is covered by the MIT license as defined in our [LICENSE](https://github.com/choderalab/asapdiscovery/blob/main/LICENSE) and adheres to the [**Developer Certificate of Origin**](https://developercertificate.org/).

### CODE REVIEW

**Comment by hmacdope** [comment]
@JenkeScheen @jthorton could I snag a review here please? :+1:

File: asapdiscovery-simulation/asapdiscovery/simulation/simulate.py [hunk 1]
```diff
@@ -235,6 +235,20 @@ def check_steps(cls, values):
             )
         return values
 
+    @property
+    def n_frames(self) -> int:
+        return self.num_steps // self.reporting_interval
+
+    @property
+    def total_simulation_time(self) -> openmm.unit.quantity.Quantity:
+        return self.num_steps * self.timestep * unit.femtoseconds
```
Comment (jthorton) [inline asapdiscovery-simulation/asapdiscovery/simulation/simulate.py:244]: Would be great to have units but until then we should update the field strings on the `VanillaMDSimulator` class to detail the intended units.

**Review by jthorton — DISMISSED** [review]
This looks great, would be good to detail the intended units of the inputs though till we use units in the schema.

**Comment by hmacdope** [comment]
Thanks for the review @jthorton. Could I snag a re-review?

**Review by kaminow — APPROVED** [review]
looks good!

**Pull request merged**